src/main/java/org/alliancegenome/curation_api/dao/base/BaseSQLDAO.java

```diff
@@ -667,10 +667,10 @@ public SearchResponse<E> findByParams(Pagination pagination, Map<String, Object>
 		if (orderByField != null) {
 			query.orderBy(builder.asc(root.get(orderByField)));
 		} else {
-			// Metamodel metaModel = entityManager.getMetamodel();
-			// IdentifiableType<E> of = (IdentifiableType<E>)
-			// metaModel.managedType(myClass);
-			// query.orderBy(builder.asc(root.get(of.getId(of.getIdType().getJavaType()).getName())));
+			// Else always order by the ID field to prevent random lists when using different page,limit combinations
+			Metamodel metaModel = entityManager.getMetamodel();
+			IdentifiableType<E> of = (IdentifiableType<E>)metaModel.managedType(myClass);
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'typecast' is not followed by whitespace.
<!-- __reviewdog__:ChA1MzQyODIxMTg1MmY5ODNiEgpjaGVja3N0eWxl -->